src/Exception/DomainAcceptsNoMail.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Egulias\EmailValidator\Exception;
+
+class DomainAcceptsNoMail extends InvalidEmail
+{
+    const CODE = 154;
+    const REASON = 'Domain accepts no mail (Null MX, RFC7505)';
```

Comment:
Is  `mail` correct or it should be `email` ?

Reply:
The title of RFC7505 is "A "Null MX" No Service Resource Record for Domains That Accept No Mail", so felt mail was more coherent.